### PR TITLE
ENCD-3889 supersedes refepi

### PR DESCRIPTION
**Pull request by jahilton**
ENCD-3889 add supersedes/superseded_by properties to ReferenceEpigenome object
Display those properties on the ReferenceEpigenome page

### CODE REVIEW

File: src/encoded/upgrade/dataset.py [hunk 1]
```diff
@@ -372,8 +372,15 @@ def dataset_18_19(value, system):
...
 @upgrade_step('experiment', '19', '20')
 def dataset_19_20(value, system):
     # https://encodedcc.atlassian.net/browse/ENCD-3974
+    return
+
+
+@upgrade_step('reference_epigenome', '14', '15')
+def dataset_20_21(value, system):
+    # https://encodedcc.atlassian.net/browse/ENCD-3889
     return
```
[the remark this answers is not in the document]
Comment (hitz) [reply src/encoded/upgrade/dataset.py]: not very strict!

File: src/encoded/schemas/reference_epigenome.json [hunk 2]
```diff
@@ -46,7 +46,19 @@
     },
     "properties": {
         "schema_version": {
-            "default": "14"
+            "default": "15"
```
[the remark this answers is not in the document]
Comment (hitz) [reply src/encoded/schemas/reference_epigenome.json]: I think the doc is wrong.

[the remark this answers is not in the document]
Comment (forresttanaka) [reply src/encoded/schemas/reference_epigenome.json]: @hitz Could we update the doc then?

**Comment by jahilton** [comment]
PR is now in ENCD-#### format
Merge commit was removed
Added to the phases in loadxl.py as @forresttanaka pointed out

**Pull request merged**